tests/lib/helpers.js

```diff
@@ -218,6 +232,13 @@ function assertUATrackingPageVar(test, key, val) {
 }
 
 
+function assertNotVisible(selector) {
```

Comment:
I'm guessing you didn't solve the problem with visibility checks in casper failing. Did you try debugging with screenshots?